apidoc/Titanium/UI/Android/ProgressIndicator.yml

```diff
@@ -44,9 +44,9 @@ properties:
 
   - name: canceledOnTouchOutside
     summary: |
-        Sets whether this dialog is canceled when touched outside the window's bounds.
+        When `cancelable` is set to `true` and this is set to `true`, the dialog is canceled when touched outside the window's bounds.
     type: Boolean
-    default: true
+    default: false
```

Comment:
Document that the `default` changed in the next version of Ti, like.... "starting in 4.0.0, this will be set to `true` by default"